packages/stems/tsconfig.json

```diff
@@ -19,6 +19,7 @@
     "noImplicitAny": true,
     "strict": true,
     "suppressImplicitAnyIndexErrors": true,
+    "ignoreDeprecations": "5.0",
```

Comment:
What's this for?

Reply:
rando, but noticed when building stems an error notice appears, we have this in @audius/common tsconfig as well. we should migrate off the depraceted tsconfig options though, which is "allowUnsafeIndexing"